test/unit/spec/v2/view-builder/internals/BaseFooter_spec.js

```diff
@@ -2,15 +2,25 @@ import BaseFooter from 'v2/view-builder/internals/BaseFooter';
 import AppState from 'v2/models/AppState';
 import Settings from 'models/Settings';
 import Link from 'v2/view-builder/components/Link';
+import { FORMS_WITHOUT_SIGNOUT, FORMS_FOR_VERIFICATION } from 'v2/ion/RemediationConstants';
 
 describe('v2/view-builder/internals/BaseFooter', function () {
 
   let testContext;
 
-  const renderFooter = (links) => {
+  const renderFooter = (links, currentFormName) => {
     const FooFooter = BaseFooter.extend({
       links,
     });
+    spyOn(AppState.prototype, 'get').and.callFake(name => {
```

Comment:
nit:
I'd prefer to unit test BaseFooter by mocking `showSignoutLinkInCurrentForm` function. Plus unit test `showSignoutLinkInCurrentForm` itself. and add testcafe for _integration_ testing.

Reply:
there are currently no unit tests for AppState, but I'll create one for this then. I won't cover the rest of the AppState model though, that would have to be done later

Reply:
Yeah, didn't mean to create unit test for other method of AppState but only the new method.